ui/app/components/clients/page/token.hbs

```diff
@@ -3,199 +3,99 @@
   SPDX-License-Identifier: BUSL-1.1
 ~}}
 
-<div class="box is-sideless is-fullwidth is-marginless is-bottomless">
-  <p class="has-bottom-margin-xl">
-    This dashboard will surface Vault client usage over time. Clients represent a user or service that has authenticated to
-    Vault.
-    <Hds::Link::Inline @href={{doc-link "/vault/docs/concepts/client-count"}}>Documentation is available here</Hds::Link::Inline>.
-    Date queries are sent in UTC.
-  </p>
-  <h2 class="title is-6 has-bottom-margin-xs">
-    {{this.versionText.label}}
-  </h2>
-  <div data-test-start-date-editor class="is-flex-align-baseline">
-    {{#if this.formattedStartDate}}
-      <p class="is-size-6" data-test-date-display>{{this.formattedStartDate}}</p>
-      <Hds::Button
-        class="has-left-margin-xs"
-        @text="Edit"
-        @color="tertiary"
-        @icon="edit"
-        @iconPosition="trailing"
-        {{on "click" (fn (mut this.showBillingStartModal) true)}}
+{{#if this.byMonthActivityData}}
+  <Clients::ChartContainer
+    @title="Entity/Non-entity clients"
+    @description="This data can be used to understand how many entity and non-entity clients are using Vault each month for this date range."
+    @timestamp={{@activity.responseTimestamp}}
+    @legend={{this.legend}}
+    @hasChartData={{true}}
+    data-test-chart="monthly total"
+  >
+    <:subTitle>
+      <h2 class="chart-title">Total monthly clients</h2>
+      <p class="chart-subtext">
+        Each client is counted once per month. This can help with capacity planning.
+      </p>
+    </:subTitle>
+
+    <:stats>
+      <StatText
+        @label="Average total clients per month"
+        @value={{this.average this.byMonthActivityData "clients"}}
```

Comment:
This total will include sync clients. 

The averages for each are shown below in the monthly new graph, so I think we should just client side sum the total clients (we need this number for the stat boxes anyway)

Reply:
Yes good call I just pulled this from the existing `monthly-usage` component without considering the updated total.